sdk/src/main/java/com/sygic/travel/sdk/di/KodeinSetup.kt

```diff
@@ -56,10 +64,13 @@ object KodeinSetup {
 	}
 
 	private val placesModule = Kodein.Module {
-		constant("places_endpoint") with API_BASE_URL + VERSION_AND_LOCALE + "/"
+		constant("places_endpoint") with "https://api.sygictravelapi.com/$VERSION_AND_LOCALE/"
+		constant("database_name") with "st-sdk-db"
 
 		bind<HeadersInterceptor>("headersIntercept") with singleton {
-			HeadersInterceptor(apiKey = instance("apiKey"), userAgent = createUserAgent(instance()))
+			HeadersInterceptor(
+				apiKey = instance("apiKey"),
+				userAgent = UserAgentUtil.createUserAgent(instance()))
```

Comment:
coding style, the closing ) should be on the new line

Reply:
:heavy_check_mark: